torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4375,10 +4454,6 @@ def method_tests():
         ('topk', (), (1, 0), 'dim_scalar', (), [1]),
         ('topk', (), (1, 0, True), 'dim_desc_scalar', (), [1]),
         ('topk', (), (1, 0, True, True), 'dim_desc_sort_scalar', (), [1]),
-        ('take', (S, S, S), (torch.LongTensor([[-3, 2], [20, 2]]),)),
```

Comment:
Removing these is great. The autograd tests for put are here:

https://github.com/pytorch/pytorch/blob/7605ce4ed86af147301721e489fc527b270f6c5e/test/test_autograd.py#L2090

Like these 'take' tests, let's verify the cases tested for 'put' are covered by the new SampleInputs and then delete these old autograd tests.

Reply:
These have already been removed in the PR.